zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -118,15 +119,60 @@ const FAKE_TESTNET_ACTIVATION_HEIGHTS: &[(block::Height, NetworkUpgrade)] = &[
 
 /// The Consensus Branch Id, used to bind transactions and blocks to a
 /// particular network upgrade.
-#[derive(Copy, Clone, Debug, Eq, Hash, PartialEq)]
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    Default,
+    Eq,
+    Hash,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    serde::Serialize,
+    serde::Deserialize,
+)]
+
 pub struct ConsensusBranchId(u32);
 
+impl ConsensusBranchId {
+    /// Return the hash bytes in big-endian byte-order suitable for printing out byte by byte.
+    ///
+    /// Zebra displays transaction and block hashes in big-endian byte-order,
+    /// following the u256 convention set by Bitcoin and zcashd.
+    fn bytes_in_display_order(&self) -> [u8; 4] {
+        let mut reversed_bytes = self.0.to_le_bytes();
+        reversed_bytes.reverse();
+        reversed_bytes
```

Comment:
Optional simplification

```suggestion
        self.0.to_be_bytes()
```

Reply:
https://github.com/ZcashFoundation/zebra/blob/main/zebra-chain/src/block/hash.rs#L32-L35